packages/serverless-api/package.json

```diff
@@ -10,7 +10,7 @@
     "build": "tsc",
     "watch": "tsc --watch",
     "build:noemit": "tsc --noEmit",
-    "docs": "typedoc --options typedoc.json",
+    "docs": "echo 'fix typedoc --options typedoc.json'; exit 1;",
```

Comment:
Is it intended that `npm run docs` will now log a command and exit with a failure (non-zero)?

Reply:
Ah, I see your note about the typedoc. I'll resolve this since you addressed it already :)

Reply:
jep so I don't forget about it :D